packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_wallets_credit_transactions.py

```diff
@@ -93,6 +93,25 @@ class TransactionClassification(str, enum.Enum):
         doc="Service run id connected with this transaction",
         index=True,
     ),
+    sa.Column(
+        "payment_transaction_id",
+        sa.String,
+        nullable=True,
+        doc="Service run id connected with this transaction",
+        index=True,
+    ),
+    sa.Column(
+        "created_at",
```

Comment:
We still have inconsistencies in names (amazing how complicated something so simple can be when working in teasm ;-) )
Indeed suffix `_at` at the API denotes datetime but we do not use it in the database. SEE packages/postgres-database/src/simcore_postgres_database/models/_common.py 

I guess at some point we should decide for one, write it in https://github.com/ITISFoundation/osparc-simcore/blob/c0db38f523fadd3f787b378c39865ff4c1e89c3c/docs/coding-conventions.md and change all

Reply:
I think we already had a discussion @pcrespov . if we start again we shall vote;) last time I checked we did go against the *_at* in the database ;)